frontend/app/views/search/_representative_file_version_cell.html.erb

```diff
@@ -1,8 +1,15 @@
 <%
   json = ASUtils.json_parse(record["json"])
   file_uri = json['representative_file_version'] ? json['representative_file_version']['file_uri'] : nil
+  file_caption = json['representative_file_version'] &&
+    json['representative_file_version']['caption'] &&
+    !json['representative_file_version']['caption'].empty? ?
+      clean_mixed_content(json['representative_file_version']['caption']) :
+      nil
```

Comment:
I wonder if rewriting this as a standard `if` statement might make this more clear. At first glance I thought `file_caption = json['representative_file_version']` might have been a typo and was meant to be a comparison with `==`, and then I got all confused on how it was all working. 😆